NOTICE.md

```diff
@@ -815,63 +971,57 @@ Java Client Runtime for AutoRest
 JsonSchemaValidator
 * Project: https://github.com/networknt/json-schema-validator
 * Source: scm:git://github.com:networknt/json-schema-validator.git
-* Maven Artifact: com.networknt/json-schema-validator/1.0.72
+* Maven Artifact: com.networknt/json-schema-validator/1.0.76
 * License: Apache-2.0
 
 Nimbus Content Type
 * Project: https://bitbucket.org/connect2id/nimbus-content-type
 * Source: scm:git:https://bitbucket.org/connect2id/nimbus-content-type.git
-* Maven Artifact: com.nimbusds/content-type/2.1
+* Maven Artifact: com.nimbusds/content-type/2.2
 * License: Apache-2.0
 
 Nimbus LangTag
 * Project: https://bitbucket.org/connect2id/nimbus-language-tags
 * Source: scm:git:https://bitbucket.org/connect2id/nimbus-language-tags.git
-* Maven Artifact: com.nimbusds/lang-tag/1.5
+* Maven Artifact: com.nimbusds/lang-tag/1.6
 * License: Apache-2.0
 
 Nimbus JOSE+JWT
 * Project: https://bitbucket.org/connect2id/nimbus-jose-jwt
 * Source: scm:git:https://bitbucket.org/connect2id/nimbus-jose-jwt.git
-* Maven Artifact: com.nimbusds/nimbus-jose-jwt/8.23
+* Maven Artifact: com.nimbusds/nimbus-jose-jwt/9.22
 * License: Apache-2.0
 
 Nimbus JOSE+JWT
 * Project: https://bitbucket.org/connect2id/nimbus-jose-jwt
 * Source: scm:git:https://bitbucket.org/connect2id/nimbus-jose-jwt.git
-* Maven Artifact: com.nimbusds/nimbus-jose-jwt/9.24.2
+* Maven Artifact: com.nimbusds/nimbus-jose-jwt/9.25
 * License: Apache-2.0
 
 Nimbus JOSE+JWT
 * Project: https://bitbucket.org/connect2id/nimbus-jose-jwt
 * Source: scm:git:https://bitbucket.org/connect2id/nimbus-jose-jwt.git
-* Maven Artifact: com.nimbusds/nimbus-jose-jwt/9.9.3
+* Maven Artifact: com.nimbusds/nimbus-jose-jwt/9.28
 * License: Apache-2.0
 
 OAuth 2.0 SDK with OpenID Connect extensions
 * Project: https://bitbucket.org/connect2id/oauth-2.0-sdk-with-openid-connect-extensions
 * Source: scm:git:https://bitbucket.org/connect2id/oauth-2.0-sdk-with-openid-connect-extensions.git
-* Maven Artifact: com.nimbusds/oauth2-oidc-sdk/9.7
+* Maven Artifact: com.nimbusds/oauth2-oidc-sdk/9.35
 * License: Apache-2.0
 
 checkstyle
 * Project: https://checkstyle.org/
 * Source: scm:git:git@github.com:checkstyle/checkstyle.git
-* Maven Artifact: com.puppycrawl.tools/checkstyle/9.0
-* License: Apache-2.0
+* Maven Artifact: com.puppycrawl.tools/checkstyle/10.0
+* License: LGPL-2.1+
```

Comment:
We would probably not use this license as foundation. The project probably offers multiple alternative licenses.